Cargo.toml

```diff
@@ -16,36 +16,41 @@ chrono = "0.4.0"
 csv = "1.0.0"
 derivative = "1"
 env_logger = "0.5"
+failure = "0.1.1"
 geo = "0.12.1"
 geo-types = "0.4.2"
 iso4217 = "0.3.2"
+lazy_static = "1.2.0"
 log = "0.4"
+minidom = "0.9.0"
+num-traits = "0.2.6"
 pretty_assertions = "0.6.1"
 proj = { version = "0.9.3", optional = true }
+regex = "1.1.2"
+reqwest = "0.9"
 rust_decimal = "1.0.1"
 serde = { version = "1", features = ["derive"] }
-structopt = "0.2"
-failure = "0.1.1"
 serde_json = "1"
-minidom = "0.9.0"
-zip = "0.3.2"
+structopt = "0.2"
 tempdir = "0.3.7"
+transit_model_procmacro = { version = "0.1.0", path = "./transit_model_procmacro" }
 walkdir = "2.1.4"
 wkt = "0.5.0"
-reqwest = "0.9"
-lazy_static = "1.2.0"
-regex = "1.1.2"
-num-traits = "0.2.6"
+zip = "0.3.2"
 
-[dependencies.transit_model_procmacro]
-path = "transit_model_procmacro"
-version = "0.1.0"
+[features]
+transxchange = []
```

Comment:
transxchange depends on proj

Reply:
Not yet in the current PR but it will, in a coming PR.